src/main/resources/application.yaml

```diff
@@ -14,5 +14,5 @@ spring:
         hbm2ddl:
           auto: create-drop
     open-in-view: false
-  config:
-    import: optional:secrets.yaml
\ No newline at end of file
+  profiles:
+    active: prod
```

Comment:
Are you sure that it should be prod, not dev?

Reply:
changed